ogr/abstract.py

```diff
@@ -89,6 +138,15 @@ def save_archive(self, filename):
         file.close()
 
 
+class GitTag:
+    def __init__(self, name: str, commit_sha: str) -> None:
+        self.name = name
+        self.commit_sha = commit_sha
+
+    def __str__(self) -> str:
+        return f"GitTag(name={self.name}, commit_sha={self.commit_sha})"
```

Comment:
TODO: Add a link to `GitTag` from `Release`.

Reply:
I've created a new issue for this so we can merge: https://github.com/packit-service/ogr/issues/74